src/sdl2_compat.c

```diff
@@ -1350,6 +1352,79 @@ SDL_LOG_IMPL(Error, ERROR)
 SDL_LOG_IMPL(Critical, CRITICAL)
 #undef SDL_LOG_IMPL
 
+/* !!! FIXME: when we override SDL_Quit(), we need to free/reset gamepad_button_swap_list */
```

Comment:
We override SDL_Quit already (for gesture stuff)

Reply:
Great, I added cleanup for that.

Reply:
Should we do that cleanup after SDL3_Quit like you do, or before?

Reply:
My intuition is that there may be events in SDL_Quit() and we might need the data there, but I'll move it beforehand for consistency with everything else for now.

Reply:
OK then